src/engraving/libmscore/select.cpp

```diff
@@ -624,7 +624,7 @@ void Selection::updateSelectedElements()
             }
         }
     }
-    Fraction stick = startSegment()->tick();
+    Fraction stick = tickStart();
```

Comment:
Also fixes crash mentioned in https://github.com/musescore/MuseScore/issues/13291